ehsm_kms_service/kubernetes/ehsm-kms.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: intel-ehsm-deployment
+  namespace: intel-ehsm
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: intel-ehsm
+  template:
+    metadata:
+      labels:
+        app: intel-ehsm
+    spec:
+      volumes:
+      - name: dev-enclave
+        hostPath:
+          path: /dev/sgx/enclave
+      - name: dev-aesmd
+        hostPath:
+          path: aesmd-socket
+      containers:
+      - name: intel-ehsm
+        image: ehsm_kms_service:latest
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /dev/sgx/enclave
+          name: dev-enclave
+        - mountPath: /var/run/aesmd
+          name: dev-aesmd
+        env:
+        - name: EHSM_CONFIG_COUCHDB_USERNAME
+          valueFrom:
+            secretKeyRef:
+              name: ehsm-secret
+              key: couch-root-username
+        - name: EHSM_CONFIG_COUCHDB_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: ehsm-secret
+              key: couch-root-password
+        - name: EHSM_CONFIG_COUCHDB_SERVER
+          valueFrom:
+            configMapKeyRef:
+              name: ehsm-configmap
+              key: database_url
+        - name: EHSM_CONFIG_COUCHDB_PORT
+          valueFrom:
+            configMapKeyRef:
+              name: ehsm-configmap
+              key: database_port
+        - name: EHSM_CONFIG_COUCHDB_DB
+          valueFrom:
+            configMapKeyRef:
+              name: ehsm-configmap
+              key: database_name
```

Comment:
can we combine the  the env variable into one?
EHSM_CONFIG_COUCHDB_DB_URL?

which the EHSM_CONFIG_COUCHDB_DB_URL=http://couch-root-username:couch-root-password@database_url:database_port?

Reply:
It's better to set it separately